context/app/static/js/components/workspaces/TemplateGrid/TemplateGrid.tsx

```diff
@@ -18,11 +18,11 @@ function TemplateGrid({
   disabledTemplates = {},
 }: TemplateGridProps) {
   return (
-    <Grid container columnSpacing={2} alignItems="stretch" sx={{ maxHeight: '625px', overflowY: 'auto' }}>
```

Comment:
This looks like a workaround to that subtle bug with MUI Grid item placement, (i.e. since MUI Grid uses flexbox, the column/row spacing props get wonky if there's any wrapping that occurs) but I could be mistaken.

Reply:
This was to adjust the way the template grid is positioned on larger pages - `columnSpacing` applies padding to the left of each of the template cards, meaning the grid was right-aligned, while `paddingX` spreads the padding equally to the left and right, so the grid is now centered.